activerecord/test/cases/adapters/postgresql/datatype_test.rb

```diff
@@ -7,7 +7,7 @@ class PostgresqlTime < ActiveRecord::Base
   # Declare attributes to get rid from deprecation warnings on ActiveRecord 6.1
   attribute :time_interval,        :string
   attribute :scaled_time_interval, :interval
-end
+end if current_adapter?(:PostgreSQLAdapter)
```

Comment:
These are for `bin/test` testing.

`bundle exec rake test` loads test files for the testing adapter, but `bin/test` always loads all test files, `:interval`, `:point`, and `:legacy_point` types are registered only for postgresql adapter, now it will raise unknown type error for sqlite3 and mysql2 adapters.

https://github.com/rails/rails/blob/098fd7f9b3d5c6f540911bc0c17207d6b48d5bb3/activerecord/Rakefile#L55-L57